docs/DeveloperGuide.md

```diff
@@ -67,9 +67,13 @@ interact with a given component through its interface rather than the concrete c
 
 ### Class Diagrams
 Command Package:
-![Class diagram for Commands](images/CommandClass.png)
+
+![Class diagram for Commands](images/CommandClassInheritance.png)
+![Class diagram for CheckInformationCommand](images/CheckInformationCommandClass.png)
+![Class diagram for CheckInformationCommand](images/PersonalTrackerCommandClass.png)
```

Comment:
i think you cannot use this kind of arrow head right? 
![image](https://github.com/user-attachments/assets/c62cf5d6-3cd2-4984-b00d-ed858da92192)

Reply:
@chewycj